cli/cli.go

```diff
@@ -45,10 +46,11 @@ func NewCommand() *cobra.Command {
 		Short:   "FirmwareUploader.",
 		Long:    "FirmwareUploader (FirmwareUploader).",
 		Example: "  " + os.Args[0] + " <command> [flags...]",
-		Args:    cobra.NoArgs,
```

Comment:
There's no need to remove this I think? :thinking:

Reply:
You are right, I'll reintroduce it